src/sign_msg_auth_token.c

```diff
@@ -57,33 +50,23 @@ static const nbgl_pageInfoLongPress_t review_final_long_press = {
 static void review_final_callback(bool confirmed) {
     if (confirmed) {
         int tx = set_result_auth_token();
-        io_seproxyhal_io_heartbeat();
         send_response(tx, true, false);
         nbgl_useCaseStatus("MESSAGE\nSIGNED", true, ui_idle);
```

Comment:
Same remark regarding mixed snake_case + camelCase

Reply:
`nbgl_useCaseStatus` is form the SDK, I would have liked a full `snake_case` but it's not up to me